### DISCUSSION TITLE
Changed pictures and delay

### DISCUSSION THREAD

**Discussion by luacomacento** [aahnik/tgcf#238] [Q&A]
Hi, first of all I would like to say how great tgcf is! Thank you for the development of this amazing tool.

I am currently facing an issue, however. The source channel that I'm forwarding from (in live mode) sometimes edit some of the messages and change the sent picture. TGCF is able to catch the edited text, but not the edited picture.

I would like to know if there is a way to solve this or if it just isn't possible or planned for future updates.

A way that I think would allow me to solve this is by having a delay between the original message and the forward, because the change usually happens after a minute or so. But as I understand it, the delay feature is only for past mode, am I correct?

Anyway, if you'd be able to share some light around this, I'd be grateful.

Thank you :)

**Comment by tissole** [comment]
This issue has been discussed #232 and a solution is planned.
